### PR TITLE
Network Performance tc's: changes for fetching worker pod logs

### PR DESCRIPTION
**Pull request by VivekThrivikraman-est**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Currently the worker pod logs cannot be accessed, this change is for making worker pods to write logs directly to /var/log on host.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

NONE
```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml [hunk 1]
```diff
@@ -20,6 +20,9 @@ spec:
       - image: gcr.io/k8s-testimages/netperfbenchmark:0.1
```
Comment (marseel) [inline clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml:20]: You will probably need to create new image and push it ;)

File: util-images/network/netperfbenchmark/cmd/main.go [hunk 2]
```diff
@@ -17,9 +17,15 @@ limitations under the License.
...
 import (
+	"flag"
+
+	"k8s.io/klog"
 	"k8s.io/perf-tests/util-images/network/netperfbenchmark/pkg/worker"
 )
 
 func main() {
+	klog.InitFlags(flag.CommandLine)
+	flag.Set("logtostderr", "false")
```
Comment (marseel) [inline util-images/network/netperfbenchmark/cmd/main.go:28]: I would prefer to move it to args, that's how we usually do it.  Example
https://github.com/kubernetes/perf-tests/blob/81c96c34e3c1f11c5fe91744ef7ce7bf44c2fe5c/clusterloader2/pkg/measurement/common/probes/manifests/ping-server-deployment.yaml#L25

File: clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml (hunk 1 above)
Comment (VivekThrivikraman-est) [reply clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml:20]: @wojtek-t could you please help in creating new image after this is committed?

File: util-images/network/netperfbenchmark/cmd/main.go (hunk 2 above)
Comment (VivekThrivikraman-est) [reply util-images/network/netperfbenchmark/cmd/main.go]: Modified.

File: clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml (hunk 1 above)
Comment (marseel) [reply clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml:20]: I can help you with it ;)

Comment (VivekThrivikraman-est) [reply clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml:20]: Ok thanks!:)

**Pull request merged**